src/snowflake/cli/_plugins/nativeapp/same_account_install_method.py

```diff
@@ -10,23 +11,12 @@
 from snowflake.cli._plugins.stage.manager import StageManager
 
 
+@dataclass
```

Comment:
Turning this to a dataclass for automatically implemented equality comparison, which is useful for tests which compare mock call args